exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,59 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class BaseAuditColumns:
+    TIMESTAMP = timestamp_column("LOG_TIMESTAMP", precision=3)
+    SESSION_ID = decimal_column("SESSION_ID", precision=20)
+    RUN_ID = decimal_column(
+        "RUN_ID",
+        precision=20,
+        comment="use POSIX_TIME(SYSTIMESTAMP()) * 1000",
+    )
+    ROWS_COUNT = decimal_column("ROWS_COUNT", precision=36)
+    QUERY_HANDLER_ID = decimal_column("QUERY_HANDLER_ID", precision=32)
+    QUERY_HANDLER_NAME = varchar_column("QUERY_HANDLER_NAME", size=2000000)
+    # QUERY_HANDLER_PHASE: TBC
+    SPAN_TYPE = varchar_column("SPAN_TYPE", size=128)
+    SPAN_ID = decimal_column("SPAN_ID", precision=32)
+    SPAN_DESCRIPTION = varchar_column("SPAN_DESCRIPTION", size=2000000)
+    OBJECT_SCHEMA = varchar_column(
+        "DB_OBJECT_SCHEMA",
+        size=128,
+        comment="Optional, can be NULL",
+    )
+    OBJECT_NAME = varchar_column(
+        "DB_OBJECT_NAME",
+        size=128,
+        comment="Contains the schema name for operations CREATE/DROP SCHEMA.",
+    )
+    OBJECT_TYPE = varchar_column("DB_OBJECT_TYPE", size=128)
+    OPERATION_NAME = varchar_column("DB_OPERATION_TYPE", size=128)
```

Comment:
not needed anymore, we can use SPAN_TYPE

Reply:
see next push.